modules/end-user-guide/pages/troubleshooting-webview-loading-error.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: PROCEDURE
+:description: Troubleshooting Webview loading error
+:keywords: user-guide, troubleshooting-webview-loading-error
+:navtitle: Troubleshooting Webview loading error
+:page-aliases: .:troubleshooting-webview-loading-error.adoc
+
+[id="troubleshooting-webview-loading-error"]
+
+= Troubleshooting Webview loading error
+
+pass:[<!-- vale RedHat.Spelling = NO -->]
+
+If you use Microsoft Visual Studio Code - Open Source in an incognito or private browser window, you can encounter an issue with Webviews. The Webview is not loaded and an error message appears: *Error loading webview: Error: Could not register service workers*.
+
+The issue affects the following browsers:
+
+* Google Chrome in Incognito mode
+* Mozilla Firefox in Private Browsing mode
+
+[WARNING]
```

Comment:
It find it a bit confusing: the warning is about Mozilla and the procedure is about Chrome.
Let me think if I can suggest some alternative.

Reply:
just to clarify: the idea is that there's no workaround for Firefox, but it exists for Chrome.